selfdrive/car/nissan/values.py

```diff
@@ -45,7 +46,6 @@ class NissanCarInfo(CarInfo):
 }
 
 FINGERPRINTS = {
-  # pylint: disable=C0301
```

Comment:
how is this handled now?

Reply:
ruff doesn't allow disabling blocks of code like pylint. I made ruff exclude E501 by putting
```# ruff: noqa: E501``` to the top of the file

only other alternative I see is adding that line to the end of every fingerprint

Reply:
don't see any regex comparing ability in ruff